build/charts/antrea/conf/antrea-controller.conf

```diff
@@ -71,3 +71,25 @@ nodeIPAM:
   # or when IPv6 Pod CIDR is not configured. Valid range is 64 to 126.
   nodeCIDRMaskSizeIPv6: {{ .nodeCIDRMaskSizeIPv6 }}
 {{- end }}
+
+ipsecSigner:
+{{- with .Values.ipsecSigner }}
+  # Enable the signer controller within the Antrea controller.
+  enable: {{ .enable }}
+  # Determines the auto approval policy of Antrea signer for IPsec certificates management.
+  # It has the following options:
+  # never:                          Controller will never auto-approve CertificateSingingRequests and they need
```

Comment:
Is there any difference between `enable=false` and `enable=true, autoApprovePolicy=never`?

Reply:
for code logic if `enable=false` the controller will not be initialized. I think this is needed for common deployment without IPsec enabled as `ipsec.yml` requires more permissions regarding RBAC.

Reply:
what's the use case of initializing the controller but not approving any CSR?

Reply:
Why controller needs to know IPsec is enabled or not, if it does not approve CSR?

Reply:
I just realized signing and approving are different, please ignore this comment.